common/cpp/react/renderer/components/rnscreens/RNSScreenComponentDescriptor.h

```diff
@@ -51,7 +51,7 @@ class RNSScreenComponentDescriptor final
       // state update.
       if (screenShadowNode.getFrameCorrectionModes().check(
               FrameCorrectionModes::Mode::FrameHeightCorrection) &&
-          compareFrameSizes(
+          !compareFrameSizes(
```

Comment:
We could consider changing the name of this function to something like `checkIsFrameSizeEqual` so that it's obvious what it returns.

Reply:
I think this is good suggestion. `compare` may suggest inequality, right? 
I'll do it in separate PR, thanks!